### ISSUE TITLE
clip-path isn't prefixed

### ISSUE THREAD

**Issue by benleibert** [LeaVerou/prefixfree#210]
What as awesome library – thank you! It seems as though clip-path isn't properly prefixed with -webkit- in Chrome, though.

-Ben Leibert

**Comment by adamclaxon** [comment]
This seems to still be an issue in Safari 10.

**Comment by LeaVerou** [comment]
Yeah, this is a known issue for properties that come from SVG. In a nutshell, -prefix-free sees that the unprefixed property is also available and thinks it doesn't have to do anything about it. It doesn't know that they don't both accept the same syntax…
